rpm/iptables.spec

```diff
@@ -1,8 +1,7 @@
 Name:       iptables
 Summary:    Tools for managing Linux kernel packet filtering capabilities
-Version:    1.8.2
+Version:    1.8.7
 Release:    1
-Group:      System/Base
 License:    GPLv2
 URL:        http://www.netfilter.org/
 Source0:    http://www.netfilter.org/projects/iptables/files/%{name}-%{version}.tar.bz2
```

Comment:
This should be just %{name}-%{version}.tar.bz2 as we do not use the upstream tarballs.

Reply:
Changed in 4d5cb7d932a61067afa064f44ac0d7d118ba5690